mr/src/main/java/org/apache/iceberg/mr/IcebergRecordReader.java

```diff
@@ -20,11 +20,10 @@
 package org.apache.iceberg.mr;
 
 import org.apache.hadoop.conf.Configuration;
-import org.apache.iceberg.DataFile;
-import org.apache.iceberg.FileScanTask;
-import org.apache.iceberg.Schema;
-import org.apache.iceberg.StructLike;
+import org.apache.iceberg.*;
```

Comment:
You need to run the whole build as I'm sure the checkstyle plugin will moan about this.